test/test_files/copy/copy_to_csv.test

```diff
@@ -9,7 +9,7 @@
 -STATEMENT load from "${DATABASE_PATH}/tinysnb_person.csv"(header=true) return *
 ---- 8
 [0:0]|0|Alice|1|True|False|35|5.000000|1900-01-01|2011-08-20 11:25:30|3 years 2 days 13:02:00|[10,5]|[Aida]|[[10,8],[6,7,8]]|[96,54,86,92]|1.731000|a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11
-[0:1]|2|Bob|2|True|False|30|5.100000|1900-01-01|2008-11-03 15:25:30.000526|10 years 5 months 13:00:00.000024|[12,8]|[Bobby]|[[8,9],[9,10]]|[98,42,93,88]|0.990000|a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a12
+[0:1]|2|Bob|2|True|False|30|5.100000|1900-01-01|2008-11-03 15:25:30.000526|10 years 5 months 13:00:00.000024|[12,8]|[Bobby]|[[8,9],[9,10]]|[98,42,93,88]|.990000|a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a12
```

Comment:
Is this 0.990000 -> .990000 change expected?

Reply:
Yes. Real values are scanned as decimals by default, and this is how the formatting works for decimals.